#! /bin/sh /usr/share/dpatch/dpatch-run
## 22-PortableGroup-compilation-fix.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix dependency problem for PortableGroup.

@DPATCH@

--- ACE_wrappers/TAO/orbsvcs/orbsvcs/PortableGroup.mpc-	2005-05-15 17:39:18.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/orbsvcs/PortableGroup.mpc	2005-05-15 17:39:29.000000000 +0200
@@ -21,9 +21,9 @@
     PortableGroup {
       miopC.cpp
       miopS.cpp
+      PortableGroup_Simple_DSC.cpp
       PortableGroupC.cpp
       PortableGroupS.cpp
-      PortableGroup_Simple_DSC.cpp
       PortableGroup
     }
   }
